s3-spark/src/test/scala/geotrellis/spark/store/s3/S3SpaceTimeSpec.scala

```diff
@@ -47,28 +38,26 @@ class S3SpaceTimeSpec
 
   lazy val bucket = "mock-bucket"
   lazy val prefix = "catalog"
-  val client = MockS3Client()
+  val client = MockS3Client.instance
```

Comment:
Funny that this doesn't have to be `@transient lazy val` because its used for local tests only.

Reply:
Ah, that's because it is used only in the following block of code: 

```
val client = MockS3Client.instance
S3TestUtils.cleanBucket(client, bucket)
registerAfterAll { () =>
  S3TestUtils.cleanBucket(client, bucket)
}
```